tests/unit-mpi/test_dry_run.py

```diff
@@ -85,7 +85,7 @@ def test_dry_run_distribute_cells(create_tmp_simulation_config_file, mpi_ranks):
         assert rank_allocation_standard['RingB'][(1, 0)] == [2]
 
     # Test redistribution
-    rank_alloc, _, _ = nd._dry_run_stats.distribute_cells_with_validation(1, 1, None)
+    rank_alloc, _, _ = nd._dry_run_stats.distribute_cells_with_validation(1, 1)
```

Comment:
very small nitpick: we decided in the past to name the unused variables like `_bau, _miao`. Not just `_, _`